users.go

```diff
@@ -87,6 +87,7 @@ type User struct {
 	PrivateProfile            bool               `json:"private_profile"`
 	SharedRunnersMinutesLimit int                `json:"shared_runners_minutes_limit"`
 	CustomAttributes          []*CustomAttribute `json:"custom_attributes"`
+	WebURL                    string             `json:"web_url"`
```

Comment:
Please place this field above `CreateAt` so it follows the ordering in the docs (which I try to use as guideline here). Thanks!

Reply:
And it seems some tests also need to be updated to make Travis pass...

Reply:
Sure, sorry about that. I've moved the field to honour the docs ordering and updated the failing tests.